assets/animations/player/movement/Run.anim.toml

```diff
@@ -25,6 +25,11 @@ if_previous_script_key = "testtt"
 action = "SetFrameNext"
 frame_index = 1
 
+[[script]]
+run_at_frame = 5
```

Comment:
```suggestion
run_at_frame = [3, 7]
```

Reply:
Added the frames where feet touch the ground in the run animation to illustrate the sound syncing nicely.